### ISSUE TITLE
SweetAlert2 Can't get error message from controller [BUG]

### ISSUE THREAD

**Issue by dirgareborn** [jeroennoten/Laravel-AdminLTE#1068]
### Describe the bug

1. update data on views  
2. to controller for validate and return with message by status
can get messages for success status, but why i can't get error status message?

in controller

```php
try {
} catch(Exception $e) {
     return response()->json([
         'status'    => 'success',
          'message'   => 'Data Jabatan Gagal Terupdate',
     ]);
}
return response()->json([
    'status'    => 'error',
    'message'   => 'Data Jabatan Sukses Terupdate',
]);
```

js

```js
var Toast = Swal.mixin({
        toast: true,
        position: 'bottom-end',
        showConfirmButton: false,
        timer: 3000
        });
    $.ajax({
        .......
        },
        success: function (response) {
                Toast.fire({
                    text: response.message,
                    icon: 'success',
                });
            },
        error: function (response) {
            Toast.fire({
                text: response.message,
                icon: 'error',
            });
        }
````

success message
[![error.png](https://i.postimg.cc/BQCC9Z6W/error.png)](https://postimg.cc/nCCmqJBT)
error message still empty
[![errors.png](https://i.postimg.cc/fbV79xvZ/errors.png)](https://postimg.cc/mcGFx1q6)
| Item    | Version
| ------- | -------
| Laravel | 
| Project | 
| OS      | 

### Additional context
Add any other context about the problem here. If you already have ideas to solve the issue, add them here or create a Pull Request (PR).

**Comment by Shidersz** [comment]
From JQuery:

> error
> Type: [Function](http://api.jquery.com/Types/#Function)( [jqXHR](http://api.jquery.com/Types/#jqXHR) jqXHR, [String](http://api.jquery.com/Types/#String) textStatus, [String](http://api.jquery.com/Types/#String) errorThrown )
A function to be called if the request fails. The function receives three arguments

The **error callback** will only be called if the request fails. In your case, you are catching errors with the **try...catch** block and always returning a response, so the ajax call won't fail with error. However, I do not undertsand why the try block is empty and why when there is an exception you are returning success status, something is wrong there...

In summary, looking at your controller, you may try something like this:

```js
var Toast = Swal.mixin({
    toast: true,
    position: 'bottom-end',
    showConfirmButton: false,
    timer: 3000
});

$.ajax({
    ...
},
success: function (response) {
    Toast.fire({
        text: response.message,
        icon: response.status,
    });
},
error: function (jqXHR, textStatus, errorThrown) {
    Toast.fire({
        text: errorThrown,
        icon: 'error',
    });
}
```

Please, next time take into considerarion this section should be used to report bugs about the package, and not to solve problems on your logic. Use **Stackoverflow** in these cases...
